lobby.go

```diff
@@ -845,6 +850,9 @@ func CreateLobby(w http.ResponseWriter, r *http.Request) {
 	clientsPerIPLimit, clientsPerIPLimitInvalid := parseClientsPerIPLimit(r.Form.Get("clients_per_ip_limit"))
 	enableVotekick := r.Form.Get("enable_votekick") == "true"
 
+	// Read wordlist according to the chosen language
+	readWordList(language)
```

Comment:
Every lobby will cause a read of the wordlist. All wordlists should be read once on startup and then never again.

Reply:
How can I do that without using the `init` function? Because I need to pass the language chosen by the user.

Reply:
actually, nevermind, this is better